tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/TaggedMetricRegistry.java

```diff
@@ -41,4 +42,6 @@
     Counter counter(MetricName metric);
 
     Map<MetricName, Metric> getMetrics();
+
+    void removeMetrics(List<MetricName> metricNameList);
```

Comment:
I think we need to adjust the API to be singular and have a return value to indicate the metric that was removed, if it existed and was successfully removed.

```
    /**
     * Removes the tagged metric with the specified metric name.
     * @param metricName metric name
     * @return the removed metric
     */
    Optional<Metric> remove(MetricName metricName);
```